model/config.go

```diff
@@ -1172,6 +1172,7 @@ type EmailSettings struct {
 	SMTPPassword                      *string `restricted:"true"`
 	SMTPServer                        *string `restricted:"true"`
 	SMTPPort                          *string `restricted:"true"`
+	SMTPServerTimeout                 *int
```

Comment:
Just a suggestion, if `SMTPServerTimeout` is defined as a `*time.Duration` instead of an `*int` then the units for the stored timeout are implied by the type, and the default defined on line 1257 can be defined as `10 * time.Seconds` or similar, to make the duration of the timeout clearer.